raco/language/myrialang.py

```diff
@@ -547,6 +548,9 @@ def __init__(self, input):
     def num_tuples(self):
         return self.input.num_tuples()
 
+    def partitioning(self):
```

Comment:
Shouldn't `partitioning` for the Producer side of an exchange operator reflect the partitioning it will see from its input (i.e., the actual partitioning of tuples on the worker where it's running)?